src/main/java/org/apache/pirk/query/wideskies/QueryInfo.java

```diff
@@ -6,9 +6,9 @@
  * to you under the Apache License, Version 2.0 (the
  * "License"); you may not use this file except in compliance
  * with the License.  You may obtain a copy of the License at
- *
- *   http://www.apache.org/licenses/LICENSE-2.0
- *
+ * <p/>
```

Comment:
The ASF header should not contain markup.
True for a number of places in the PR.